src/test/regress/expected/multi_tenant_isolation.out

```diff
@@ -798,84 +798,6 @@ $$;
 RESET citus.enable_metadata_sync;
 CREATE EVENT TRIGGER abort_drop ON sql_drop
    EXECUTE PROCEDURE abort_drop_command();
-\c - postgres - :master_port
--- Disable deferred drop otherwise we will skip the drop and operation will succeed instead of failing.
-SET citus.defer_drop_after_shard_split TO OFF;
-SET ROLE mx_isolation_role_ent;
-SET search_path to "Tenant Isolation";
-\set VERBOSITY terse
-SELECT isolate_tenant_to_new_shard('orders_streaming', 104, 'CASCADE', shard_transfer_mode => 'block_writes');
-WARNING:  command DROP TABLE is disabled
-WARNING:  command DROP TABLE is disabled
-WARNING:  command DROP TABLE is disabled
-WARNING:  command DROP TABLE is disabled
-WARNING:  command DROP TABLE is disabled
-WARNING:  command DROP TABLE is disabled
-WARNING:  failed to clean up 6 orphaned shards out of 6 after a isolate_tenant_to_new_shard operation failed
-ERROR:  command DROP TABLE is disabled
```

Comment:
This removed test seems like it might test something useful. Can you check git history why it was added? And then maybe replace it with something else.